src/components/form_elements/SelectInput.js

```diff
@@ -1,14 +1,20 @@
-import React from 'react';
+import React, { useContext } from 'react';
 
 import { ALL, COMPLETED, UNCOMPLETED, PROGRESS } from '../../constants';
+import ThemeContext from '../../contexts/ThemeContext';
 
 const SelectInput = ({ setStatus }) => {
+  const theme = useContext(ThemeContext);
   const statusHandler = e => {
     setStatus(e.target.value);
   };
   return (
-    <div className="select">
-      <select onChange={statusHandler} name="todos" className="filter-todo">
+    <div className={`select ${theme}`}>
+      <select
+        onChange={statusHandler}
+        name="todos"
+        className={`filter-todo ${theme}`}
```

Comment:
Pls write CSS with selectors (https://www.w3schools.com/cssref/css_selectors.asp) and parent-child combination specifically (https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator) 
`div.className > select { //css here }`
In that way your style will be more structured

Reply:
fixed